README.md

```diff
@@ -70,10 +76,6 @@ module.exports = {
 };
 ```
 
-### Memory limit
-
-Use `-m` to change `pg_dump` output memory limit (e.g: `512`)
-
```

Comment:
So, this is now obsolete, right?

Reply:
Correct. I'm debating leaving the option in so people's scripts don't break, what do you think? maybe a message saying this is obsolete?